### ISSUE TITLE
Paper adjustments

### ISSUE THREAD

**Issue by bjnorfolk** [bjnorfolk/summerproject#4]
APJ:
I have a few questions:
* I think it's important to include the maximum range in [Sr/Fe], [Ba/Fe] in the cannon training set. Presumably it's something like +0.5, which is why you have to identify them as flux residuals rather than just getting the numbers out of the Cannon.
* I think you should show another figure that includes the MIKE spectra + syntheses of Sr and Ba. There's a surprising number of people who (unreasonably, IMHO) don't think low-res spectra can measure individual abundances like this. For those people I think showing the followup spectra would be convincing. Along those lines, it would be good to measure the C/Na/Tc abundance for those spectra just to confirm those elements.
* One thing I wonder about is microturbulence. This is a "fake" stellar parameter that's supposed to account for unmodeled turbulence in the star, so it's something that I'd naively expect to be different in e.g. AGB vs RG at fixed logg. For strong saturated lines (Sr, Ba, Na) this can significantly affect the abundance, and I don't think it's usually included as a parameter in the Cannon (or even in APOGEE sometimes?). Not sure this is worth discussing in this paper though.
* For the Na, I'd guess that NLTE effects could also play a role. This maybe was discussed in deCastro+16 but worth bringing up again, especially for giants vs FGK dwarfs.
* The other thing is that in low resolution spectra I wonder if the Na line saturates and so you don't get enough of a flux enhancement even with an abundance enhancement. It would be helpful to have some extra numbers (e.g. how big is the size of the Na enhancement, and whether you'd expect it to make a significant flux difference).
* Uncertainties from abundance analysis to be added to table 1 ?
* figure showing lines+syntheses for mag/mike

**Comment by bjnorfolk** [comment]
Kevin:
-Should the 'B' in "Barium stars" always be capitalized?
-In Table 1, stars that have a check in the column "Ba II & Sr II"
    do not have checks in both "Ba II" and "Sr II".  That is, the
    adopted notation does not conform to the standard definition of
    Boolean AND logic.
-Four candidates were observed with Magellan/MIKE, yet the paper only
    mentions two stars.  What became of the other two stars?  If those
    stars didn't present the anomalous strontium and/or barium abundances,
    then that needs to be stated.  If that is true, were they used to
    improve the candidate selection algorithm?  And is the number of
    candidates mentioned in the abstract the number that remains after
    those improvements were made?

    All stars observed:
    J080502.41+053810.4
    J083514.72-054848.0
    J083847.03+224111.2
    J091628.34+025934.8
 Brodie: So yeh I thought we did 4, should I just change that section to only have 2 ? 

-For the stars with TGAS parallaxes, how well does the logg from Ho et
    al. (2017) compare to a logg inferred from using the measured parallax,
    photometric magnitudes, and isochrones?`

-I think the most likely explanation for a lack of AGB stars in
     the barium star sample is a lack of AGB stars in the APOGEE sample
     used to train the Ho et al. (2017) data-driven model.  If that is
     true, then this analysis does not require unaccounted selection
     biases in previous studies towards luminous intrinsic barium stars.
-I would argue that Figure 3 should not
     be over-interpreted given the fact that plotted tracks don't do a
     good job matching the observations in the range -0.5 < [Fe/H] < 0.0.
-I find it odd that the conclusion advocates follow-up observations
     with high-resolution spectrographs in order to precisely measure a
     full suite of neutron-capture abundances when the high-resolution
     spectra available for this paper were not analyzed to measure a
     full suite of neutron-capture abundances (only strontium and barium)

**Comment by bjnorfolk** [comment]
Anna:
-There are many more LAMOST spectra available now, in DR3. I think it would be worth acknowledging that, saying how many spectra they have (1.5 million more than in DR2, I think!) and justifying why you decided to stick with DR2.
Brodie: is this because Cannon only fitted to DR2 ect ?

-where did the 0.2 dex systematic error floor come from ?

**Comment by andycasey** [comment]
Are you able to address at least some of these without my input, or are all of these points things you want input on?

**Comment by bjnorfolk** [comment]
I'm not sure if you're just referring to Anna's or all the issues. I'm just
struggling with whether a lot of the issues are relevant or can be
justified with one sentence, such as the LAMOST DR3 comment by Anna, like
was it because Anna fitted to DR2 ? and therefore we can ignore that
comment. Or even the 0.2 dex systematic error, like I don't even know how
you got that ?
Most of Alex's comments were on further telescope stuff that I think is not
relevant but I wasn't sure ?
And kind of the same trend with Kevins

**Comment by andycasey** [comment]
> - I think it's important to include the maximum range in [Sr/Fe], [Ba/Fe] in the cannon training set. Presumably it's something like +0.5, which is why you have to identify them as flux residuals rather than just getting the numbers out of the Cannon.

[Sr/Fe] and [Ba/Fe] are not included in The Cannon training set as labels, but the training set probably includes stars of [Ba,Sr/Fe] up to +0.5, so this is near the level you would need before we would detect them as enhancements.

We need to look in flux residuals because [Ba/Fe] and [Sr/Fe] are not labels in the cannon model.

> - I think you should show another figure that includes the MIKE spectra + syntheses of Sr and Ba. There's a surprising number of people who (unreasonably, IMHO) don't think low-res spectra can measure individual abundances like this. For those people I think showing the followup spectra would be convincing. Along those lines, it would be good to measure the C/Na/Tc abundance for those spectra just to confirm those elements.

This would be good to include. No time for report though.

> - One thing I wonder about is microturbulence. This is a "fake" stellar parameter that's supposed to account for unmodeled turbulence in the star, so it's something that I'd naively expect to be different in e.g. AGB vs RG at fixed logg. For strong saturated lines (Sr, Ba, Na) this can significantly affect the abundance, and I don't think it's usually included as a parameter in the Cannon (or even in APOGEE sometimes?). Not sure this is worth discussing in this paper though.

Not included in The Cannon,  it is included in APOGEE. Microturbulence values were taken from stellar parameter relations used in the Gaia-ESO survey.

> - For the Na, I'd guess that NLTE effects could also play a role. This maybe was discussed in deCastro+16 but worth bringing up again, especially for giants vs FGK dwarfs.

It does. This is what you are arguing: low logg values in previous analyses have resulted in systematically higher [Na/Fe], no?

> - The other thing is that in low resolution spectra I wonder if the Na line saturates and so you don't get enough of a flux enhancement even with an abundance enhancement. It would be helpful to have some extra numbers (e.g. how big is the size of the Na enhancement, and whether you'd expect it to make a significant flux difference).

Quite possible. You should mention this as a caveat.

> - Uncertainties from abundance analysis to be added to table 1 ?

If there's  room, include them.

> - figure showing lines+syntheses for mag/mike


> -Should the 'B' in "Barium stars" always be capitalized?

Should it? Or can you just refer to them as 'barium stars' when referring to the class of objects

> -In Table 1, stars that have a check in the column "Ba II & Sr II"
> do not have checks in both "Ba II" and "Sr II". That is, the
> adopted notation does not conform to the standard definition of
> Boolean AND logic.

Correct. Surely you can explain why.

> -Four candidates were observed with Magellan/MIKE, yet the paper only
> mentions two stars. What became of the other two stars? 

I thought we only observed 2 of your stars. Did we observe 2 or 4?
